metrics/metrics/src/main/java/io/helidon/metrics/ExemplarService.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.metrics;
+
+/**
+ * Behavior for supporting exemplars in metrics histograms.
+ */
+public interface ExemplarService {
+
+    /**
+     * Defayult priority for an {@code ExemplarService} found by the service loader without an explicit {@code @Priority}
```

Comment:
Default priority

Reply:
Fixed. (I wish I could blame this on my Southern heritage, but I have none.)